packages/visualizations-react/src/reactify/ReactImpl.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useRef, useLayoutEffect } from 'react';
+import {
+    SvelteComponent,
+    ComponentProps,
+    ComponentConstructorOptions,
+} from '@opendatasoft/visualizations'; // we export from the main package to avoid having different versions of svelte
+
+/* Your ComponentProps type definition here */
+function reactifySvelte<C extends SvelteComponent>(
+    Component: new (options: ComponentConstructorOptions) => C // Correct constructor signature
+) {
+    return (props: ComponentProps<C>) => {
+        const svelteComponentRef = useRef<C | null>(null);
+        const mountRef = useRef<HTMLDivElement | null>(null);
+        // FIXME: temporary props dispatch to match the old implem
+        const { style, ...componentProps } = props;
+
+        useLayoutEffect(() => {
+            if (mountRef?.current) {
+                const component = new Component({
+                    target: mountRef.current,
+                    props: componentProps,
+                });
+                svelteComponentRef.current = component;
+            }
+
+            return () => {
+                svelteComponentRef.current?.$destroy();
+            };
+            // We especially don't want to react to props to avoid creating a new component each render
+            // eslint-disable-next-line react-hooks/exhaustive-deps
+        }, [mountRef]);
+
+        useEffect(() => {
+            if (svelteComponentRef?.current) {
+                svelteComponentRef.current.$set({ ...componentProps });
+            }
+        }, [componentProps]);
+
+        // FIXME: ideally, we want to
+        return <div ref={mountRef} style={style} />;
```

Comment:
I saw your PR about style.
Can't we add a class to this div to indicate this is the start of a ods-dataviz-sdk visualization. 
So this is common to all viz and can be used to target it form parent context. 

I can see something like this: 
```tsx
// ReactImpl.tsx
function reactifySvelte<C extends SvelteComponent>(
    Component: new (options: ComponentConstructorOptions) => C, // Correct constructor signature
    className: string
) {
// internal logic
return <div ref={mountRef} className={className} />;
}
```
```ts
// index.ts
export const Chart = reactifySvelte(_Chart, "ods-visualization ods-visualization-chart");
// ... //
```

Reply:
I thought I had `className='ods-reactify'` already. It did at some point and I think it's the easiest yes. We could  indeed be more specific :thup: Anyways I like the notify the user that its' a "technical" node.

Reply:
ReactImpl could add the `ods-dataviz-reactify` class (like you said, to notify about this technical node) and each svelte visual component could have a `ods-visualization ods-visualization-{type}` class